From d7e9381dc2e409904c78650e71af6b12a191af85 Mon Sep 17 00:00:00 2001
From: Peter Yin <peter.yin@quantatw.com>
Date: Wed, 17 Sep 2025 15:39:16 +0800
Subject: [PATCH] ARM: dts: aspeed: harma: use power-monitor instead of
 power-sensor

lore:https://lore.kernel.org/all/20250917101828.2589069-3-peteryin.openbmc@gmail.com/

Revise the node name from "power-sensor" to "power-monitor"
in the Harma device tree to follow Device Tree naming convention.

Signed-off-by: Peter Yin <peter.yin@quantatw.com>
---
 .../dts/aspeed/aspeed-bmc-facebook-harma.dts  | 20 +++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
index ef5705f89c00..8b9ff836cf02 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-harma.dts
@@ -210,25 +210,25 @@ pwm@5e {
 		reg = <0x5e>;
 	};
 
-	power-sensor@40 {
+	power-monitor@40 {
 		compatible = "ti,ina238";
 		reg = <0x40>;
 		shunt-resistor = <1000>;
 	};
 
-	power-sensor@41 {
+	power-monitor@41 {
 		compatible = "ti,ina238";
 		reg = <0x41>;
 		shunt-resistor = <1000>;
 	};
 
-	power-sensor@44 {
+	power-monitor@44 {
 		compatible = "ti,ina238";
 		reg = <0x44>;
 		shunt-resistor = <1000>;
 	};
 
-	power-sensor@45 {
+	power-monitor@45 {
 		compatible = "ti,ina238";
 		reg = <0x45>;
 		shunt-resistor = <1000>;
@@ -291,25 +291,25 @@ pwm@5e {
 		reg = <0x5e>;
 	};
 
-	power-sensor@40 {
+	power-monitor@40 {
 		compatible = "ti,ina238";
 		reg = <0x40>;
 		shunt-resistor = <1000>;
 	};
 
-	power-sensor@41 {
+	power-monitor@41 {
 		compatible = "ti,ina238";
 		reg = <0x41>;
 		shunt-resistor = <1000>;
 	};
 
-	power-sensor@44 {
+	power-monitor@44 {
 		compatible = "ti,ina238";
 		reg = <0x44>;
 		shunt-resistor = <1000>;
 	};
 
-	power-sensor@45 {
+	power-monitor@45 {
 		compatible = "ti,ina238";
 		reg = <0x45>;
 		shunt-resistor = <1000>;
@@ -401,7 +401,7 @@ power-monitor@40 {
 		reg = <0x40>;
 	};
 
-	power-sensor@45 {
+	power-monitor@45 {
 		compatible = "ti,ina238";
 		reg = <0x45>;
 		shunt-resistor = <500>;
@@ -558,7 +558,7 @@ imux28: i2c@0 {
 			#address-cells = <1>;
 			#size-cells = <0>;
 			reg = <0>;
-			power-sensor@20 {
+			power-monitor@20 {
 				compatible = "mps,mp5990";
 				reg = <0x20>;
 			};
-- 
2.53.0-Meta

